### PR TITLE
Gherkin Scripts Helm Release Scenarios - [ODC-3022]

### PR DESCRIPTION
**Pull request by gajanan-more**
**Fixes:** https://issues.redhat.com/browse/ODC-4168

**Problem:** Gherkin Scripts Design for Helm Release epic ODC-3022

**Solution:** These are the gherkin scripts designed covering all the scenarios related to Helm Release epic ODC-3022

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/Helm/helm-Chart-Version-ODC-3022.feature [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    When user checks the Helm Charts checkbox
+    And user searches for the 'Node-ex-k' helm chart
+    And user clicks on the 'Node-ex-k' helm chart card
+    And user will see the information of all the chart versions together
+    And user clicks on the Install Helm Chart button
+    Then user will see the dropdown of chart version
+
+
+@regression
+Scenario: Update the chart version to see the data lost
```
Comment (debsmita1) [inline frontend/packages/dev-console/integration-tests/features/Helm/helm-Chart-Version-ODC-3022.feature:44]: The alert modal will pop up only if there is(are) unsaved change(s) in the yaml editor/ dynamic form and the user tries to update the chart version.

File: frontend/packages/dev-console/integration-tests/features/Helm/helm-Chart-Version-ODC-3022.feature [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    Given user is at the Install Helm Chart page
+    When user clicks on the Chart Versioon dropdown menu 
+    And user selects the different chart version
+    Then modal will get popped up
+    And modal will have the old and new chart versions
+    And modal will have the warning of data lost
+
+
+@regression, @manual
+Scenario: README should be updated with updating chart version
```
Comment (debsmita1) [inline frontend/packages/dev-console/integration-tests/features/Helm/helm-Chart-Version-ODC-3022.feature:54]:
```suggestion
Scenario: README should be updated when chart version is updated
```

File: frontend/packages/dev-console/integration-tests/features/Helm/actions-on-HelmRelease.feature [hunk 3]
```diff
@@ -1,6 +1,7 @@
 Feature: Helm Chart
     User will be able to perform the actions on the helm releases
 
+
```
[the remark this answers is not in the document]
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/Helm/actions-on-HelmRelease.feature:4]: Updated the file with creating helm charts and performing actions on it.

**Pull request merged**